client/ayon_core/tools/push_to_project/control.py

```diff
@@ -97,23 +103,23 @@ def set_source(self, project_name, version_id):
 
         self._src_folder_entity = folder_entity
         self._src_folder_task_entities = task_entities
-        self._src_product_entity = product_entity
-        self._src_version_entity = version_entity
-        if folder_entity:
+        self._src_product_entities = product_entities
+        self._src_version_entities = version_entities
+        if folder_entity and len(list(version_entities)) == 1:
```

Comment:
Why only when one version is selected?
```suggestion
        if folder_entity:
```